configs/razor/subnets/README.md

```diff
@@ -0,0 +1,54 @@
+# Projecs Based on MMRAZOR
+
+There are many research works and pretrained models built on MMRAZOR. We list some of them as examples of how to use MMRAZOR slimmable models for downstream framework. As the page might not be completed, please feel free to contribute more efficient mmrazor-models to update this page.
+
+## Description
+
+This is an implementation of MMRAZOR Searchable Backbone Application, we provide detection configs and models for MMRAZOR in MMYOLO.
+
+### Backbone support
+
+#### NAS model
+
+- \[x\] [AttentiveMobileNetV3](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/configs/_base_/nas_backbones/attentive_mobilenetv3_supernet.py)
+- \[x\] [SearchableShuffleNetV2](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/configs/_base_/nas_backbones/spos_shufflenet_supernet.py)
+- \[x\] [SearchableMobileNetV2](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/configs/_base_/nas_backbones/spos_mobilenet_supernet.py)
+
+## Usage
+
+### Prerequisites
+
+- [MMRAZOR v1.0.0rc2](https://github.com/open-mmlab/mmrazor/tree/v1.0.0rc2) or higher (dev-1.x)
+
+### Training commands
+
+In MMYOLO's root directory, run the following command to train the model:
+
+```bash
+CUDA_VISIBLE_DEVICES=0,1,2,3 PORT=29500 ./tools/dist_train.sh configs/razor/subnets/yolov5_s_spos_shufflenetv2_syncbn_8xb16-300e_coco.py
```

Comment:
Do we have command for single GPU for training ? Cuz most user do not have mul GPU for training. I suggest add a command for it.